x-pack/plugins/spaces/public/management/edit_space/edit_space_general_tab.test.tsx

```diff
@@ -407,6 +407,73 @@ describe('EditSpaceSettings', () => {
     expect(navigateSpy).toHaveBeenCalledTimes(1);
   });
 
+  it('submits the disabled features list when the solution view is undefined', async () => {
+    const features = [
+      new KibanaFeature({
+        id: 'feature-1',
+        name: 'feature 1',
+        app: [],
+        category: DEFAULT_APP_CATEGORIES.kibana,
+        privileges: null,
+      }),
+    ];
+
+    const spaceToUpdate = {
+      id: 'existing-space',
+      name: 'Existing Space',
+      description: 'hey an existing space',
+      color: '#aabbcc',
+      initials: 'AB',
+      disabledFeatures: [],
+      solution: undefined,
+    };
+
+    render(
+      <TestComponent>
+        <EditSpaceSettingsTab
+          space={spaceToUpdate}
+          history={history}
+          features={features}
+          allowFeatureVisibility={true}
+          allowSolutionVisibility={true}
+          reloadWindow={reloadWindow}
+        />
+      </TestComponent>
+    );
+
+    // update the space visible features
+    await act(async () => {
```

Comment:
I usually only wrap with `act` the part of the code that has side effects (here for example the `await userEvent.click(feature1Checkbox);`). That's how it is documented in the docs https://react.dev/reference/react/act#dispatching-events-in-tests and in most tests I've seen.
Leaving the assertions outside.

Reply:
240afff1f92